src/main/java/uk/gov/hmcts/dts/fact/services/admin/list/AdminAreasOfLawService.java

```diff
@@ -53,6 +56,16 @@ public AreaOfLaw createAreaOfLaw(final AreaOfLaw areaOfLaw) {
         return new AreaOfLaw(areasOfLawRepository.save(createNewAreaOfLawEntityFromModel(areaOfLaw)));
     }
 
+    public void deleteAreaOfLaw(final Integer areaOfLawId) {
```

Comment:
Does this need a transactional annotation?

Based on the docs, I think the CRUD method is already transactional: https://docs.spring.io/spring-data/data-jpa/docs/current/api/org/springframework/data/jpa/repository/support/SimpleJpaRepository.html#deleteById-ID-

Reply:
If the only thing you are doing here is deleting, then no. If you have more than one operation on the database, then I would say to dig into it a big more. 

For example if we are saving, deleting, etc on one method, then a transactional operation may be valid, but in this case, I think deletes are transactional by default